src/app/manager-dashboard/manager-dashboard.component.ts

```diff
@@ -50,7 +50,7 @@ export class ManagerDashboardComponent implements OnInit {
       this.message = 'Access restricted to admins';
     }
     // Check parent if configuration exists
-    this.couchService.post(`communityregistrationrequests/_find`, findOneDocument('name', this.userService.get().name),
+    this.couchService.post(`communityregistrationrequests/_find`, findOneDocument('name', this.userService.getConfig().name),
```

Comment:
we should check code instead of name as that is what we have been using to identify planet